lambda/events_id/get/index.js

```diff
@@ -2,104 +2,127 @@
 /*
 *  Zak Brinlee
 *  AD 440 - ActoKids practicum
-*  Sprint 3 task - Logging relevant data from GET event by id function
-*  This Lambda function is currently on my personal AWS account
+*  Sprint 5 task - Fix GET by event_id and prep for prod demo
 */
 
 // aws-sdk is used for accessing aws services through libraries 
 const AWS = require('aws-sdk');
 
 // Lambda function to handle GET requests for specific Events in the ActoKids DynamoDB events table by the event_id key.
-// This function will need to have the handling for PUT and DELETE requests during later sprints
+
+// this package is needed to validate parameters that are uuids
+const isUUID = require('is-uuid');
 
 // handler object created by Lambda and serves as the entry point that AWS Lambda uses to execute your function code. 
 // event is the header and parameters from the request
-exports.handler = async (event, context) => {
+exports.handler = async (event, context, callback) => {
     
     // Console log to CloudWatch for Lambda function start
-    console.log("ak-api-zak-lambda GET event by event_id function started.");
+    console.log("ActoKids GET event by event_id function started.");
     
     // Insantiate new instance of DynamoDB object with specified api version. 
     const db = new AWS.DynamoDB({ apiVersion: '2012-10-08'});
```

Comment:
db is assigned a value but never used.  This line and associated comment can be deleted.

Reply:
fixed